lib/runtime2/threepio/capabilities/generate-domcument/Operator.ts

```diff
@@ -0,0 +1,147 @@
+
+import fs from 'fs';
+import { APP_ROOT_ID } from '.';
+
+function appendHtml(htmlstr: string, element: Element) {
+  const tempDiv = document.createElement('div');
+  tempDiv.innerHTML = htmlstr;
+  Array.from(tempDiv.childNodes).forEach(childNode => {
+    element.appendChild(childNode);
+  });
+}
+
+export class Operator {
```

Comment:
```suggestion
export class DOMOperator {
```
`Operator` 太通用了，改成 `DOMOperator` 更合适